.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
This clearly wasn't here to start with, and you need to run `git rm` on your node_modules so that this isn't so polluted with code you didn't write.